R/zzz.R

```diff
@@ -0,0 +1,17 @@
+register_mlr3 = function() {
+  mlr_resamplings = utils::getFromNamespace("mlr_resamplings", ns = "mlr3")
+  mlr_resamplings$add("variable_size_train_cv", function() ResamplingVariableSizeTrainCV$new())
```

Comment:
maybe remove this line? or add another line for ResamplingSameOtherCV?
I would like to encourage users to use ResamplingSameOtherSizesCV, rather than the others, so I am inclined to only add that to the mlr_resamplings.

Reply:
also instead of `function() ResamplingVariableSizeTrainCV$new()` can this just be `ResamplingVariableSizeTrainCV$new` ?

Reply:
Ok, I removed the variable_size_train_cv. 

Also, even just adding `ResamplingVariableSizeTrainCV` works, so I did that.

Reply:
great thanks